assets/js/link-feedback.js

```diff
@@ -0,0 +1,13 @@
+const feedbackIssueLinks = document.querySelectorAll('.feedback-issue');
```

Comment:
Looks like we are adding listeners to everything. I would propose dropping all of the listeners that are being added throughout the code and instead add one giant click-listener to the document and then check the `target` within the listener's callback to determine what to do. As in
```
document.addEventListener('click', e => {
    const { target } = e;
    if (target.matches('.feedback-issue')) return gtag('event', 'submit_issue_click');
    if (target.matches('.feedback-edit')) return return gtag('event', 'edit_page_click');
    if (target.matches('.copy-label')) return copyLabel(target);
    if (target.matches('.copy-curl-label')) return copyLabel(target, true);
});

Reply:
That's a good idea!